app/src/main/java/a3/com/convo/activities/LoginActivity.java

```diff
@@ -34,17 +34,18 @@
 import java.util.HashMap;
 import java.util.List;
 
+import a3.com.convo.Constants;
 import a3.com.convo.Models.Page;
 import a3.com.convo.R;
 
 public class LoginActivity extends AppCompatActivity {
 
-    LoginButton loginButton;
-    CallbackManager callbackManager;
-    Activity context;
+    private LoginButton loginButton;
+    private CallbackManager callbackManager;
+    private Activity context;
```

Comment:
Do we need to save context as a member variable? Its usually a bad coding practice to save activity as member variable. It leads to memory leaks.